test/labeling/model/test_label_model.py

```diff
@@ -88,11 +93,141 @@ def test_augmented_L_construction(self):
         self.assertEqual(L_aug[1, j], 1)
         self.assertEqual(L_aug[2, j], 1)
 
-    def test_with_deps(self):
-        for seed in range(self.n_iters):
-            np.random.seed(seed)
-            data = SingleTaskTreeDepsGenerator(self.n, self.m, k=self.k, edge_prob=1.0)
-            self._test_label_model(data, test_acc=False)
+    def test_conditional_probs(self):
+        label_model = LabelModel(k=2, verbose=False)
+
+        L = np.array([[1, 2, 1], [1, 2, 1]])
+        label_model._set_constants(L)
+        label_model._set_dependencies(deps=[])
+        label_model._generate_O(L)
+        label_model.inv_form = False
+        label_model._set_class_balance(class_balance=[0.6, 0.4], Y_dev=None)
+        label_model._init_params()
```

Comment:
```python
        label_model._set_constants(L)
        label_model._set_dependencies(deps=[])
        label_model._generate_O(L)
        label_model.inv_form = False
        label_model._set_class_balance(class_balance=[0.6, 0.4], Y_dev=None)
        label_model._init_params()
```

is repeated quite a bit (or some form of it). Can we abstract that out to an internal function?